packages/elements-dev-portal/src/containers/StoplightProject.tsx

```diff
@@ -129,7 +130,10 @@ const StoplightProjectImpl: React.FC<StoplightProjectProps> = ({
     elem = (
       <NodeContent
         node={node}
-        Link={Link}
+        // Link={({ children, ...props }) => {
+        //   return <a href={props.to}>{children}</a>;
+        // }}
```

Comment:
I think that wasn't supposed to be here too

Reply:
removed :)